internal/metrics/kms_decrypter.go

```diff
@@ -68,18 +68,18 @@ func decryptKMS(kmsClient kmsiface.KMSAPI, ciphertext string) (string, error) {
 	params := &kms.DecryptInput{
 		CiphertextBlob: decodedBytes,
 	}
-	response, err := kmsClient.Decrypt(params)
+	response, err := kmsClient.Decrypt(context.TODO(), params)
 
 	if err != nil {
 		logger.Debug("Failed to decrypt ciphertext without encryption context, retrying with encryption context")
 		// Try with encryption context, in case API key was encrypted using the AWS Console
 		params = &kms.DecryptInput{
 			CiphertextBlob: decodedBytes,
-			EncryptionContext: map[string]*string{
-				encryptionContextKey: &functionName,
+			EncryptionContext: map[string]string{
+				encryptionContextKey: functionName,
 			},
 		}
-		response, err = kmsClient.Decrypt(params)
+		response, err = kmsClient.Decrypt(context.TODO(), params)
```

Comment:
This is kinda a nit pick, but I think we wanna use `context.Background()`. We can then also use the same context for both of these kms client calls. It won't change any of the functionality, so really this is just a nit, but it would allow us to do cool things like add timeouts in the future.